pkg/util/ecs/diagnosis.go

```diff
@@ -62,12 +58,10 @@ func diagnoseECSTags() error {
 func diagnoseFargate() error {
 	client, err := ecsmeta.V2()
 	if err != nil {
-		log.Debugf("error while initializing ECS metadata V2 client: %s", err)
-		return err
+		return log.Errorf("error while initializing ECS metadata V2 client: %w", err)
```

Comment:
You probably meant instead:

```suggestion
		return fmt.Errorf("error while initializing ECS metadata V2 client: %w", err)
```

Reply:
Thanks for addressing this, no need for a full approval from me